app/views/schools/partnerships/index.html.erb

```diff
@@ -0,0 +1,53 @@
+<% content_for :before_content, govuk_back_link(
+  text: "Back",
+  href: schools_cohort_path("2021"))
+%>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-m">2021</span>
+    <h1 class="govuk-heading-l">Sign up with a training provider</h1>
+    <% if @partnership %>
+    <h3 class="govuk-heading-m">Partnership details</h3>
+    <dl class="govuk-summary-list">
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          Delivery partner
+        </dt>
+        <dd class="govuk-summary-list__value">
+          <%= @partnership.delivery_partner.name %>
+        </dd>
+      </div>
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          Lead provider
+        </dt>
+        <dd class="govuk-summary-list__value">
+          <%= @partnership.lead_provider.name %>
+        </dd>
+      </div>
+    </dl>
+
+    <p>
+    <!-- TODO: Add link to challenge partnership -->
+    If this doesn't look right you can <%= govuk_link_to "report that your school has been signed up incorrectly", "#" %>
+    </p>
+    <% else %>
+    <p class="govuk-body">You need to sign a contract with a training provider so they can deliver your programme.</p>
+    <p class="govuk-body">You can find out more about what they’re offering and contact them directly:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li><%= govuk_link_to "Ambition Institute", "https://www2.ambition.org.uk/l/330231/2021-02-24/4mvx6", class: "govuk-link--no-visited-state" %></li>
+      <li><%= govuk_link_to "Best Practice Network", "https://bestpracticenet.co.uk/early-career-framework", class: "govuk-link--no-visited-state" %></li>
+      <li><%= govuk_link_to "Capita with lead academic partner the University of Birmingham", "https://www.capita.com/expertise/supporting-teachers-in-early-career", class: "govuk-link--no-visited-state" %></li>
+      <li><%= govuk_link_to "Education Development Trust", "https://www.educationdevelopmenttrust.com/ecf", class: "govuk-link--no-visited-state" %></li>
+      <li><%= govuk_link_to "Teach First", "https://www.teachfirst.org.uk/early-career-framework", class: "govuk-link--no-visited-state" %></li>
+      <li><%= govuk_link_to "UCL Institute of Education", "https://www.ucl.ac.uk/ioe-early-career-framework", class: "govuk-link--no-visited-state" %></li>
```

Comment:
Do we want noopener noreferer on these?

Reply:
Do we need them if we don't have `target: "_blank"`?

Reply:
I mean, these urls will open in the current tab, and as I understand it noreferer and noopener refer to the case when you open a new tab.

Reply:
Maybe the answer is "we want them to open in new tab" and then yes, yes we do those things too :)

Reply:
@ethanmills I can't see any reason why we might need that.